cmd/package-builder/package-builder.go

```diff
@@ -26,6 +26,19 @@ func runVersion(args []string) error {
 	return nil
 }
 
+// Adapted from
```

Comment:
Was not sure if you would want to add this to the public API of one of the packages, or just have a slight bit of duplication.

Reply:
I am totally agnostic if we expose it here. Awesome that you did the work